ui/scss/component/_stripe-card.scss

```diff
@@ -433,7 +433,7 @@ pre {
   width: 284px;
   box-shadow: unset;
   color: #32325d;
-  background-color: white;
+  background-color: white !important;
```

Comment:
Padding issues. Doesn't happen in production, so something must have changed in the branch, but couldn't find where.

<img width="406" alt="image" src="https://user-images.githubusercontent.com/64950861/185602114-96197732-532b-463a-a3e4-c0e4e5a1745f.png">